src/main/java/de/retest/web/selenium/AutocheckingCheckNamingStrategy.java

```diff
@@ -21,6 +21,15 @@
 	 */
 	String getUniqueCheckName( String action, WebElement target, Object... params );
 
+	/**
+	 * Return a string that is a unique identifier in the current test.
+	 *
+	 * @param action
+	 *            The action to use when creating a semantic name (e.g. "back").
+	 * @return The unique identifier.
+	 */
+	String getUniqueCheckName( String action );
```

Comment:
I think you could reuse the other Javadoc:

```java
/**
 * @see #getUniqueCheckName( String, WebElement, Object... )
 */
```

Reply:
Actually I changed the example ("navigate-back" instead of "click").